tck/features/SkipLimitAcceptance.feature

```diff
@@ -67,6 +67,7 @@ Feature: SkipLimitAcceptanceTest
 
 
   Scenario: LIMIT with an expression that does not depend on variables
+    Given an empty graph
```

Comment:
This should also be possible with `any graph`

Reply:
The assertion that we get a count of 2 does not hold for `any graph`.

Reply:
I think it does hold, since we create 3 nodes in the init step, then match for any nodes and limit to 2, the count is always going to be 2, regardless of how many nodes were in the graph, or what shape they had.

The only way this would not hold is if we were to take constraints into account. However, constraints are still considered out of scope for the TCK (and openCypher in general), so this is a non-argument. And either way, if openCypher adopted constraints the way they are implemented in Neo4j, the creations in the init step would still be possible because there is no label on the nodes.

So I insist that `any graph` is fine :)

Reply:
You are right.